contents/blog/sql-for-analytics.md

```diff
@@ -0,0 +1,134 @@
+---
+title: The basics of SQL for analytics
+date: 2023-08-31
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/happy-hog.png
+featuredImageType: full
+category: Engineering
+tags:
+ - Guides
+ - Explainers
+---
+
+SQL is a programming language to help you interact with data. One of SQL’s most important use cases is analytics. In analytics, there are many types of data and ways to slice that data. SQL provides the depth and customizability to get insights from it, answer questions, and make data-driven decisions.
```

Comment:
I dont think we need the "SQL is a programming language to help you interact with data". It's quite off putting If Im honest. As an engineer, its common knowledge what it is, and reading that line makes me feel that either this blog is not for engineers, or is written for SEO (and not to inform)

```suggestion
One of SQL’s most common use cases is analytics. There are many way to slice and interpret your analytics data. SQL provides the depth and customizability to get insights from it, answer questions, and make data-driven decisions.
```